articles/design/using-auth0-with-multi-tenant-apps.md

```diff
@@ -9,50 +9,48 @@ contentType: concept
 useCase: strategize
 ---
 
-# Using Auth0 with Multi-Tenant Applications
+# Using Auth0 to Secure Your Multi-Tenant Applications
 
-In this article, we'll discuss (at a high-level) how Auth0 can help you manage users for your multi-tenant applications.
+This article provides a high-level overview of how Auth0 can help you to secure your multi-tenant applications.
 
-If you're planning to build a multi-tenant solution, please [contact our Sales or Professional Services](https://auth0.com/?contact=true) teams.
+[Multi-tenancy](https://en.wikipedia.org/wiki/Multitenancy) is when a single instance of software runs on a server that is accessible to multiple groups of users. In fact, Auth0 itself is a multi-tenant application; your collection of applications, settings, and connections is called a "tenant". Note that this document is **not about using multiple Auth0 tenant(s)**, but is about using Auth0 to secure your own multi-tenant application.
 
-## What's multi-tenant?
+## Auth0 and multi-tenancy
 
-[Multi-tenancy](https://en.wikipedia.org/wiki/Multitenancy) is when a single instance of software runs on a server that is accessible to multiple groups of users.
+There are several ways you can handle multi-tenant applications with Auth0. If you intend to use a single Auth0 tenant for all your customers you can handle your multi-tenancy needs with one of the following approaches:
 
-When working with multi-tenant software, you can serve multiple customers from a single application instance running on one server (or pool of servers). This contrasts with single-tenant software, where you serve each customer with a dedicated software instance running on dedicated servers. In summation:
+* [Use multiple connections](#use-multiple-connections)
+* [Identify different tenants by application](#identify-tenants-by-application)
+* [Store tenant details in app_metadata](#store-tenant-details-in-app_metadata)
+* [Use separate Auth0 tenants](#create-separate-auth0-tenants-for-each-customer)

```

Comment:
* [Use multiple connections](#use-multiple-connections)
* [Identify each of your tenants with its own application](#identify-tenants-by-application)
* [Store tenant details in `app_metadata`](#store-tenant-details-in-appmetadata)

Note that I moved the last into its own category -- we discourage it, so I thought I'd set it off visually

Reply:
I am not sure I agree with splitting it out, though, because it's still a valid option, and now it doesn't seem like the last in a list of options, it feels like it was important enough to break out on its own.